src/main/kotlin/net/ccbluex/liquidbounce/features/command/commands/client/CommandVClip.kt

```diff
@@ -0,0 +1,49 @@
+package net.ccbluex.liquidbounce.features.command.commands.client
+
+import net.ccbluex.liquidbounce.features.command.Command
+import net.ccbluex.liquidbounce.features.command.CommandException
+import net.ccbluex.liquidbounce.features.command.builder.CommandBuilder
+import net.ccbluex.liquidbounce.features.command.builder.blockParameter
+import net.ccbluex.liquidbounce.utils.client.chat
+import net.ccbluex.liquidbounce.utils.client.regular
+import net.ccbluex.liquidbounce.utils.client.variable
+import net.minecraft.client.MinecraftClient
+
+object CommandVClip {
+
+    fun createCommand(): Command {
+        return CommandBuilder
+            .begin("vclip")
+            .parameter(
+                blockParameter("distance")
+                    .required()
+                    .build()
+            )
+            .handler { command, args ->
+                val y = (args[0] as String).toDoubleOrNull()
+                val player = MinecraftClient.getInstance().player
+
+                if (y == null) {
+                    chat(regular("Invalid distance. Please enter a valid number."))
```

Comment:
It would be nice to use command.result there as well so the result can be translated.